csharp/generate/generate.sh

```diff
@@ -55,14 +55,15 @@ echo "generating sdk"
 #java -jar swagger-codegen-cli.jar swagger-codegen-cli help
 java -jar openapi-generator-cli.jar generate \
     -i $swagger_file \
-    -g csharp \
+    -g csharp-netcore \
     -o $sdk_output_folder \
     -c $config_file \
     -t $gen_root/templates \
 	--type-mappings dateorcutlabel=DateTimeOrCutLabel \
   --type-mappings double=decimal
 
-rm -rf $sdk_output_folder/.openapi-generator
+rm -rf $sdk_output_folder/.openapi-generator/
+rm -rf $sdk_output_folder/src/Lusid.Sdk.Test/
```

Comment:
Why have you added in this line to remove the Tests?

Reply:
its the autogenerated stub tests that get created, there isn't an option to not generate them (like some of the other generators) so explicitly deleting